README.md

```diff
@@ -1,32 +1,355 @@
-# Lesson Title
+# Modifying Objects
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Review the concept of reference types
+- Practice cloning objects
+- Practice modifying objects
+- Write pure functions that act on objects
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+So far we've learned how to access information from objects, both directly using
+dot or bracket notation, and dynamically using a `for...in` loop. In this
+lesson, we'll learn how to copy and modify objects. We'll also learn how to
+modify objects _nondestructively_, so we can keep our functions pure.
 
-## Topic 1
+## Review of Reference Types
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+You may recall that we learned in a previous lesson that arrays are what is
+known in JavaScript as _reference_ types, meaning that a reference to an array
+(e.g., a variable name) is pointing to a location in memory rather than the
+value of the array itself. This means that, if you copy an array using the
+assignment operator (`=`) and make a change to the original array, the same
+change will be made to the copy (and vice versa):
 
-## Topic 2
+```js
+myArray = ["one", "two", "three"];
```

Comment:
Maybe define `myArray` and `clone` with `const` just for consistently demonstrating best practices?